File: Logistics.Web/ClientApp/src/app/models/item-group.ts
import { FundingAccount } from './funding-account';
import { ItemGroupApproval } from './item-group-approval';
import { ItemGroupCategory } from './item-group-category';
import { Order } from './order';
import { Request } from './request';
import { RequestItem } from './request-item';
import { Site } from './site';
import { User } from './user';

export class ItemGroup {
  id: number;
  fundingAccountId: number;
  itemGroupCategoryId: number;
  requestId: number;
  siteId: number;
  userId: number;
  fundingAccount: FundingAccount;
  itemGroupCategory: ItemGroupCategory;
  request: Request;
  site: Site;
  user: User;
  itemGroupApprovals: ItemGroupApproval[];
  orders: Order[];
  requestItems: RequestItem[];
}

File: Logistics.Data/ItemGroup.cs
﻿using System.Collections.Generic;

namespace Logistics.Data
{
    public class ItemGroup
    {
        public int Id { get; set; }
        public int FundingAccountId { get; set; }
        public int ItemGroupCategoryId { get; set; }
        public int RequestId { get; set; }
        public int SiteId { get; set; }
        public int UserId { get; set; }

        public FundingAccount FundingAccount { get; set; }
        public ItemGroupCategory ItemGroupCategory { get; set; }
        public Request Request { get; set; }
        public Site Site { get; set; }
        public User User { get; set; }

        public List<ItemGroupApproval> ItemGroupApprovals { get; set; }
        public List<Order> Orders { get; set; }
        public List<RequestItem> RequestItems { get; set; }
    }
}
